google/ggrc-core
# GGRC-4311 Implement GDrive interaction code on BE

# Issue description
Implement GDrive interaction code on BE.

# Steps to test changes
- attach evidence file to Audit, Assessment, Control and check that all work as before.
- check import/export flow.
- integration and unit tests.

# Solution description
Move gdrive API manipulation to the BE.
Reimplement file name postfixes.
Run ‘gdrive file copy’ flow in GGRC Document obj creation.

# Sanity checklist




# PR Review checklist
